src/Compilers/Core/Portable/CommandLine/ErrorLogger.cs

```diff
@@ -42,21 +42,17 @@ public ErrorLogger(Stream stream, string toolName, string toolFileVersion, Versi
             _writer.WriteArrayStart("runs");
             _writer.WriteObjectStart(); // run
 
-            WriteToolInfo(toolName, toolFileVersion, toolAssemblyVersion);
+            _writer.WriteObjectStart("tool");
+            _writer.Write("name", toolName);
+            _writer.Write("version", toolAssemblyVersion.ToString());
+            _writer.Write("fileVersion", toolFileVersion);
+            _writer.Write("semanticVersion", toolAssemblyVersion.ToString(fieldCount: 3));
+            _writer.Write("language", culture.Name);
```

Comment:
I believe technically, I can make custom CultureInfos with any name, but as you say, there's nothing else we can do. In practice, this will be from the built-in CultureInfos that do have the correct names.